### ISSUE TITLE
Minor quibble with Readme

### ISSUE THREAD

**Issue by benwilson512** [joakimk/toniq#7]
Hey!

Looks like a cool library. As a minor quibble though about your readme example, you have

```
  def perform(to: to, subject: subject, body: body) do
    # do work
  end
```

It is generally considered to not be a good idea to pattern match on keyword lists as they're order dependent. 

While `Toniq.enqueue(SendEmailWorker, to: "info@example.com", subject: "Hello", body: "Hello, there!")` would work `Toniq.enqueue(SendEmailWorker, subject: "Hello", to: "info@example.com", body: "Hello, there!")` would not, and that's generally confusing.

Am I correct in reading that perform takes just one argument?

**Comment by joakimk** [comment]
Yeah, I've noticed that problem and the readme could certainly be improved. Contributions welcome :)

We could maybe update the examples use use maps?

The `enqueue` takes a list of arguments, and because of that `perform` does too. So you can probably pass in just about anything there (as long as it can be serialized, so no Pids, etc).
